File: backend/src/main/java/demo/application/backend/model/AirQualityIndex.java
package demo.application.backend.model;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class AirQualityIndex {

	private int numericlevel;		// 28
	private String verbalLevel;		// Good
	private String description;		// Air quality is good. A perfect day for a walk!
	private String publicationInfo;	// Tehran Published at 07:30
	private List<PolutionIndex> polutionIndexes;
}

File: frontend/src/app/model/air-quality-index.ts
import { PolutionIndex } from "./polution-index";

export interface AirQualityIndex {
  numericlevel: number;		// 28
	verbalLevel: string;		// Good
	description: string;		// Air quality is good. A perfect day for a walk!
	publicationInfo: string;// Tehran Published at 07:30
	polutionIndexes: PolutionIndex[];
}
